### DISCUSSION TITLE
Bullet point overview of workflow

### DISCUSSION THREAD

**Discussion by percyfal** [NBISweden/Training-Tech-shorts#30] [General]
I jotted down a bullet point overview of the desired workflow. Don't know if it helps but I felt I needed to write it up in my own words to get a grip of the process. Parts of the list could be added to the preface page (https://nbisweden.github.io/Training-Tech-shorts/) or similar.

# Workflow

1. teacher lesson
   - make branch =topic=
   - make lesson file
   - add key points
   - demonstrate tool
     
2. learner notes
   - for each key point learner:
     - creates new branch from =topic=
     - makes a draft pull request
     - adds notes
     
3. learner review
   - request review from another learner
   - update until satisfied
     
5. teacher review
   - click on "Ready for review"
   - select teacher from dropdown list of reviewers
   - teacher and learner iterate review until good
     
6. teacher merges PR

**Comment by mahesh-panchal** [comment]
This is a nicer way to layout the action points. I think it would be worth restructuring the steps in this way on the index.qmd.
